datanode/api/trading_data_v2.go

```diff
@@ -116,6 +117,16 @@ func (t *TradingDataServiceV2) ListAccounts(ctx context.Context, req *v2.ListAcc
 		return nil, formatE(ErrInvalidPagination, err)
 	}
 
+	if req.Filter != nil {
+		if err := VegaIDsSlice(req.Filter.MarketIds).Ensure(); err != nil {
+			return nil, formatE(err, errors.New("one or more market id is invalid"))
+		}
+
+		if err := VegaIDsSlice(req.Filter.PartyIds).Ensure(); err != nil {
```

Comment:
I'm not sure if this will work because `network` is a valid partyID to filter on:
https://api.n06.testnet.vega.xyz/api/v2/accounts?filter.partyIds=network